lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -136,7 +140,7 @@
                         required: true,
                         refreshPageOnSave: true,
                         helpMessage:
-                            gettext('The language used for this site. The site is currently available in a limited number of languages.'),
+                            gettext('The language used throughout this site. The site is currently available in a limited number of languages.'),
```

Comment:
Using upstreams wording: https://github.com/Stanford-Online/edx-platform/blob/merge/upstream/08-20/lms/static/js/student_account/views/account_settings_factory.js#L79
Nothing interpolated so no need to call `interpolate_text`